### ISSUE TITLE
APE 13: Zenodo record needs updating

### ISSUE THREAD

**Issue by pllim** [astropy/astropy-APEs#110]
https://github.com/astropy/astropy-APEs/pull/106#issuecomment-2377800643

**Comment by eteq** [comment]
Just for the record, I have a suspicion the owner of this Zenodo entry is Steve Crawford, who was the original first author of the APE. I'll try to confirm that with him via out-of-band discussion.

**Comment by pllim** [comment]
Thanks! While you are at it, any chance Steve also owner of #108 ?

**Comment by pllim** [comment]
Still an issue as of 2024-10-21

**Comment by eteq** [comment]
Steve is  *not* the owner, but is thinking it's possible it's Nick Earl, which might explain why Zenodo can't reach him since it would be at a different email.  Will continue trying to track it down...

**Comment by pllim** [comment]
@nmearl ,  do you own this Zenodo record? Please advise. Thanks!

**Comment by nmearl** [comment]
I am not the owner. However, looking at the [record metadata](https://zenodo.org/api/records/1117943), it shows that the owner is user 818. Looking at the [api response for that user](https://zenodo.org/api/users/818), it looks like it's @kelle.

**Comment by pllim** [comment]
Amazing! You are in my books as Zenodo Sherlock now. Thanks!

**Comment by kelle** [comment]
Should I go through a process for changing ownership of the record?

**Comment by pllim** [comment]
Yes, ideally the ownership should transfer to Astropy account. Thanks!

**Comment by kelle** [comment]
Oh, I found a new feature where I can share the record with another user!  I've just given the Astropy account permission to manage the record. Let's see if that works.

**Comment by pllim** [comment]
That works, @kelle ! I hope I did it correctly.

**Comment by pllim** [comment]
Thanks, all!
